src/electron-api.d.ts

```
@@ -235,6 +235,15 @@ interface IElectronAPI {
   scanClosedVSCodeSessions: (activeSessionIds: string[]) => Promise<any[]>;
   refreshSessionPreview: (sessions: any[]) => Promise<Record<string, { lastUserMessage: string; lastAssistantMessage: string }>>;
   openClaudeSession: (sessionId: string, projectPath: string, isActive: boolean, activePid?: number, customTitle?: string) => void;
+  /** Resume the given saved-list members that are not running; staggered, skips report why. */
+  openSessionListMembers: (
+    members: { sessionId: string; project: string; accountLabel?: string; title?: string }[],
```

ESLint

[error] Replace `·sessionId:·string;·project:·string;·accountLabel?:·string;·title?:·string` with `⏎······sessionId:·string;⏎······project:·string;⏎······accountLabel?:·string;⏎······title?:·string;⏎···`

(prettier/prettier)